containers/unit_tests/cuda/TestCuda_BitSet.cpp

```diff
@@ -1,47 +0,0 @@
-
-/*
-//@HEADER
-// ************************************************************************
-//
```

Comment:
Just for fun, I'd love to know what Sandia legal would say about license headers in generated code.